libmuscle/python/libmuscle/manager/test/test_profile_database.py

```diff
@@ -0,0 +1,137 @@
+from unittest.mock import patch
+
+from libmuscle.manager.profile_database import ProfileDatabase
+from libmuscle.manager.profile_store import ProfileStore
+from libmuscle.planner.planner import Resources
+from libmuscle.profiling import (
+        ProfileEvent, ProfileEventType, ProfileTimestamp)
+
+from ymmsl import Operator, Port, Reference
+
+import pytest
+
+from pathlib import Path
+
+
+@pytest.fixture
+def db_file(tmp_path) -> Path:
+    with patch('libmuscle.manager.profile_store._SYNCHED', True):
```

Comment:
For context (no need to change anything): you can also use [pytest's `monkeypatch.setattr`](https://docs.pytest.org/en/6.2.x/monkeypatch.html). Then you don't need to put all the logic inside a `with` statement.

Reply:
Oh, interesting. I'm going to have a look at that when I have more time.